workshops/forms.py

```diff
@@ -1036,6 +1037,32 @@ class Meta:
         }
 
 
+class TrainingRequestUpdateForm(forms.ModelForm):
+    person = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.PersonLookup,
+        label='Matched Person',
+        required=False,
+        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget,
+    )
+
+    class Meta:
+        model = TrainingRequest
+        exclude = ()
```

Comment:
This is a form for admins, and admins should be able to edit all fields. It's much different situation than `TrainingRequestForm` which is accessible to trainees and where inclusion of unwanted fields is risky.